media/css/mozorg/home/home-firefox-mr106-promo.scss

```diff
@@ -0,0 +1,99 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+$image-path: '/media/protocol/img';
+
+@import '~@mozilla-protocol/core/protocol/css/includes/lib';
+@import '~@mozilla-protocol/core/protocol/css/components/logos/wordmark';
+@import '~@mozilla-protocol/core/protocol/css/components/logos/wordmark-product-firefox';
+
+.mr106-home-hero {
+    .mzp-c-split-bg {
+        background-color: $color-violet-60;
+    }
+
+    // Override Protocol spacing
+    &.mzp-t-split-nospace {
+        padding-top: 0;
+        padding-bottom: 0;
+    }
+
+    // Override Protocol wordmark
+    .mzp-c-wordmark.mzp-t-wordmark-lg.mzp-t-product-firefox {
+        background-image: url('/media/img/home/2022/logo-word-hor-white.svg');
+        background-position: center top;
+        margin-left: auto;
+        margin-right: auto;
+
+        @media #{$mq-md} {
+            background-position: left top;
+            margin-left: 0;
+            margin-right: 0;
+        }
+    }
+
+    // Override Protocol button
+    .mzp-c-button.mzp-t-product {
+        background-color: $color-white;
+        border-color: $color-white;
+        color: $color-black;
+
```

Comment:
Suggestion (non-blocking): should button copy be centre aligned?

![image](https://user-images.githubusercontent.com/400117/196691844-a61d18ee-6179-47dd-a5f7-b7eeea9b31af.png)

Reply:
This seems to be an existing issue with the button styling in Protocol. We don't explicitly declare `text-align` and when a button is just a link it inherit from its parent. `button` elements have `text-align: center` in the browser styles but links don't. I'll add it here but it's something that should really be fixed upstream in Protocol. I'll file an issue there.